### DISCUSSION TITLE
What's the difference between '@sharkdp/fd' and 'sharkdp/fd'?

### DISCUSSION THREAD

**Discussion by yusiwen** [zdharma-continuum/zinit#662] [Q&A]
The example of wiki show these two commands to install the plugin sharkdp/fd, one with the '@' sign and the other without. What's the difference?

Thanks

**Comment by pschmitt** [comment]
There's none really for that particular project.

The `@` notation is only relevant for projects whose names clash with zinit options.

If you wanted to install a repo named `null/null` for example this would clash with the `null` ice. To still be able to install this without confusing the parser you'd use `@null/null`.
